app/views/frame-add-new-user.jade

```diff
@@ -42,3 +42,5 @@ div(fullscreen)
         include partials/error-status-bar-detailed
         include partials/success-status-bar
         include partials/warning-status-bar
+        footer.tight.relative(ng-show="neo4j.clusterRole")
+          small Note: User management events do not propagate when running a core-edge cluster
```

Comment:
Small note: perhaps this can be simplified to "User management events do not propagate across a cluster."